Haruby-iOS/Sources/Feature/Input/InputViewReactor.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  InputViewReactor.swift
+//  Haruby-iOS
+//
+//  Created by Seo-Jooyoung on 10/13/24.
+//
+
+import UIKit
+import ReactorKit
+import RxSwift
+
+final class InputViewReactor: Reactor {
+    enum Action {
+        case toggleDetailButton
+        case toggleDatePicker
+        case selectTransactionType(String)
+        case tapAddDetailTransactionButton // 너무 긴가 ..
+    }
```

Comment:
사용자의 Action을 정의하는 부분이라 case명을 동사로 변경 부탁드립니다
```suggestion
    enum Action {
        case detailTransactionButtonTapped
        case datePickerTapped
        case transactionTypeButtonTapped(String)
        case addingTransactionButtonTapped // 너무 긴가 ..
    }
```

근데 위 네이밍 예시도 다 ButtonTapped로 마무리되서... 다같이 의논해봐여